_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,254 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom.
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'title-require'
+---
+
+## Applicability
+
+The rule applies to each `meta <element>` with a `name` attribute whose value is set to [`viewport`](https://www.w3.org/TR/css-device-adapt-1/#viewport-meta) and has a `content` attribute.
+
+## Expectation
+
+For each test target, the [`content`](https://www.w3.org/TR/html52/document-metadata.html#dom-htmlmetaelement-content) attribute, whose value is mapped to a list of property/value pairs in a user-agent specific manner, does not:
+
+- specify the property `user-scalable` with a value of `no`; nor
+- specify the property `maximum-scale` with a value of less than 2
+
+## Assumptions
+
+- The [page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) has [visible](#visible) [content](https://www.w3.org/TR/WCAG21/#dfn-content).
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 1.4.4: Resize text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text)
+- [The `meta <element>`](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The `meta name="viewport" <element>` does not define the `maximum-scale` and `user-scalable` values.
+
+```html
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width" />
+	</head>
+	<body>
+		<p>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
```

Comment:
Also, I'm not sure we need that much garbage in the examples (make them bigger than needed). `Lorem ipsum` should be enough.